src/HashSetHashMap/HashMap/RansomNote/RansomNote.java

```diff
@@ -0,0 +1,39 @@
+package HashSetHashMap.HashMap.RansomNote;
+
+import org.junit.Test;
+
+import java.util.PriorityQueue;
+import java.util.Queue;
+
+
+/**
+ * Ransom Note | Leetcode #0383 | Easy

```

Comment:
I wouldn't agree that this is a very popular question, but perhaps we can add a list of 'other practice questions for each of the sections as well. 
<img width="865" alt="Screen Shot 2021-09-04 at 8 42 42 AM" src="https://user-images.githubusercontent.com/40924388/132074704-3de8cca3-c948-43db-ab84-45e60bb4c934.png">